pyaerocom/aeroval/obsentry.py

```diff
@@ -75,6 +75,9 @@ class ObsEntry(BaseModel):
     only_superobs : bool
         this indicates whether this configuration is only to be used as part
         of a superobs network, and not individually.
+    is_bulkfraction: bool
+        If true numerator and denominator are colocated separately, before the fraction is calculated.
+        For this to work, the numerator and denominator need to be given in obs_aux_requires
```

Comment:
Is `obs_aux_requires` involved? I think you are using `bulk_options` instead

Reply:
fixed